srcpkgs/hikari/template

```diff
@@ -33,5 +36,9 @@ pre_build() {
 }
 
 post_install() {
+	if [ "$build_option_setuid" ]; then
+		# must be setuid for direct session management
+		chmod u+s ${DESTDIR}/usr/bin/hikari
+	fi
```

Comment:
Hikari installs its executable setuid root by default, so this change is superfluous.

Regardless of whether setuid or nosetuid becomes the "standard" behavior, or people make a convincing case for preserving this build option, Hikari needs to be tested with seat management to make sure it may be launched without setuid root when seatd or elogind are used.

Reply:
So there is an actual reason why the setuid bit isn't set. In that case I will undo this.

Regarding the setuid build option, I support disabling it by default as well, but @ericonr was of a different opinion, so this should probably be discussed further.

Reply:
I've tested hikari with elogind before and it works. The same should apply to seatd, in theory. I think the `hikari-unlocker` binary still needs to be setuid, but hikari's build system already does this.

If the setuid build option goes away, should the hikari template have something like `chmod u-s ${DESTDIR}/usr/bin/hikari` to remove setuid from the binary? Otherwise, hikari would be inconsistent with the other compositors.

Reply:
That really depends on whether hikari itself (not hikari-unlocker) is able to run without suid... the project's README doesn't mention session management at all, which means someone has to actually test it